docs/design-proposals/multi-tenancy/multi-tenancy-access-control.md

```diff
@@ -18,27 +18,54 @@ The key changes to each authenticator is that it needs to retrieve the tenant in
 
 The existing X509 certificate authenticator reads "/CN" and "/O" from cert subject, and map them to user name and groups respectively. 
 
-To make the authenticator tenancy-aware, "/O" will be mapped to tenant while "/OU" (Organization Unit) will be mapped to groups. 
+Below is a sample certificate request with the new format:
 
-Below is a sample cert:
+```text
+openssl req -new -key demo.pem -out demo-csr.pem -subj "/CN=user/O=app1/O=app2"
```

Comment:
Got confused. Isn't this the old format?

Reply:
fixed